js/web/components/HomePage.js

```diff
@@ -41,16 +43,17 @@ const HomePage = () => {
             className={classes.sectionHeader}
           >
             <Typography variant="body1" align="left">
-              New Releases
+              Highlights 
+              <span> 
+                <Button
+                  onClick={() => resetQueueWithPlaylist(releasesRecent.highlights.map(release => release.releasePubkey))}
+                >
+                  <PlayCircleOutlineOutlinedIcon sx={{ color: "black" }} />
```

Comment:
these playicons icons in the headers should get an explicit `paddingTop: 3px` to line up w the copy